app/BUILD.bazel

```diff
@@ -553,7 +553,7 @@ kt_android_library(
         ":snap_helper",
         ":view_models",
         "//third_party:androidx_appcompat_appcompat",
-        "//third_party:androidx_core_core-ktx",
```

Comment:
I think you may have misread my earlier comment. This one is still needed otherwise the build will break.

Reply:
Done.